luminaire/exploration/data_exploration.py

```diff
@@ -133,6 +133,9 @@ def add_missing_index(self, df=None, freq=None):
 
         import pandas as pd
 
+        # Adding a group by logic for duplicate index
+        df = df.groupby(df.index).mean()
```

Comment:
Nit: Should we be throw some info error if the data has duplicate index? Just want to make sure that we are not silently doing this and the users do not know about this in their data.

Reply:
This is a common issue with input data that we have encountered and we will update in the documentation about how we infer if any duplicate data is present.

Reply:
ok, so you should add the doc update/changes in this MR too.

Reply:
I will raise a separate MR for the docs as that will need good amount of changes since we have added new logics. This info will be added over there.

Reply:
Doc updates should go together.